webhook_to_fedora_messaging/config/config.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+import sys
+from os import environ
+
+import toml
+
+
+def return_flaskapp_config_data():
```

Comment:
This function name seems a bit long, no? Do you expect to have other functions to return some other configuration? If not, something shorter like `get_config()` would work. I'm sure you'll enjoy typing less ;-)